src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java

```diff
@@ -43,7 +43,7 @@
       effectTags = {OptionEffectTag.UNKNOWN},
       help =
           "Specify how spawn actions are executed by default."
-              + "'standalone' means run all of them locally."
+              + "'standalone' means run all of them locally and it disables sandboxing for tests."
```

Comment:
Not only for tests, I would change this to:

'standalone' means run all of them locally without any kind of sandboxing